docs/acceptance-criteria/uacs-tbtc-v2.adoc

```diff
@@ -0,0 +1,291 @@
+  approved is deemed valid, then: +
+  - the sortition pool should get unlocked +
+  - the DKG result should be approved and a new wallet should be created in the
+  wallets contracts based on a candidate group (minus inactive/disqualified
+  members) +
+  - the DKG result accepter should receive reimbursement for the gas spent on
+  the DKG submit and DKG approve transactions, paid from the DAO-funded ETH pool
+  to the notifier’s address (in the same tx); there should be a limit on the
+  refunded gas price (`max_gas_refund_price`) (if the amount of gwei paid for
+  gas in the DKG result submission is bigger than this value, the submitter will
+  be reimbursed `max_gas_refund_price x gas_used`) +
+  - operators marked as inactive/disqualified during DKG protocol execution
+  should be punished by being excluded from earning rewards for a fixed amount
+  of time (`sortitionPoolRewardsBanDuration`) (in the same tx).
+
+// The below DKG timeout UACs are describing how timeouts would work if the
+// procedure for submitting DKG result worked as it works currently for Random
+// Beacon (at first one operator can submit, after
+// `resultSubmissionEligibilityDelay` another is added to the list of operators
+// eligible to publish the result, and so on...). This is what initial
+// implementation of a DKG fo ECDSA looks like, but there are ongoing
+// discussions whether this is the best approach. We may end up not having
+// timeouts at all...
+
+| Wallet lifecycle - wallet creation - DKG timeout
+| The DKG submission timeout equals the group size multiplied by the number of
+  blocks for a member to become eligible to submit the DKG result plus the
+  number of blocks covering for the time of DKG generation. The timer should
+  start when the first member becomes eligible (the moment when DKG was
+  requested). The timer gets reset when a valid DKG result challenge is
+  submitted.
+
+| Wallet lifecycle - wallet creation - DKG timeout
+| Anyone can send a transaction reporting DKG timeout.
+
+| Wallet lifecycle - wallet creation - DKG timeout
+| Upon noticing the DKG timeout notification, the DKG contract should determine
+  its validity.
+
+| Wallet lifecycle - wallet creation - DKG timeout
+| When determining the validity of the DKG timeout notification, the following
+  aspects should be taken into account: +
+  - does it reference the existing DKG request? (if not, then notification
+  invalid) +
+  - has the DKG timeout passed? (if not, then notification invalid) +
+  - is the sortition pool in a locked state (if not - meaning somebody already
+  unlocked it - then notification invalid).
+
+| Wallet lifecycle - wallet creation - DKG timeout
+| If the DKG timeout notification was deemed justified, then: +
+  - the pool should be unlocked +
+  - the fixed amount reward (in T) (`sortitionPoolUnlockingReward`) should be
+  sent from the DAO-funded ETH pool to the notifier.
```

Comment:
Will we have a reward or reimbursement here? This case is not mentioned in the https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-1.adoc#transaction-incentives.